Language/Functions/Zero, Due, MKR Family/analogReadResolution.adoc

```diff
@@ -16,11 +16,12 @@ subCategories: [ "Zero, Due & MKR Family" ]
 
 [float]
 === Description
-analogReadResolution() is an extension of the Analog API for the Arduino Due, Zero and MKR Family.
+analogReadResolution() is an extension of the Analog API for the Zero, Due, MKR family, Nano 33 (BLE and IoT) and Portenta.
 
 Sets the size (in bits) of the value returned by `analogRead()`. It defaults to 10 bits (returns values between 0-1023) for backward compatibility with AVR based boards.
 
-The *Due, Zero and MKR Family* boards have 12-bit ADC capabilities that can be accessed by changing the resolution to 12. This will return values from `analogRead()` between 0 and 4095.
+The *Zero, Due, MKR family and Nano 33 (BLE and IoT)* boards have 12-bit ADC capabilities that can be accessed by changing the resolution to 12. This will return values from `analogRead()` between 0 and 4095.
+The Portenta H7 has a 16 bit ADC, which will allow values between 0 and 65535.
```

Comment:
```suggestion
The *Zero, Due, MKR family and Nano 33 (BLE and IoT)* boards have 12-bit ADC capabilities that can be accessed by changing the resolution to 12. This will return values from `analogRead()` between 0 and 4095. +
The *Portenta H7* has a 16 bit ADC, which will allow values between 0 and 65535.
```
- Line break to improve readability.
- Bold "Portenta H7" to match the formatting of the previous sentence.